### ISSUE TITLE
User changed from Admin to User retained admin rights

### ISSUE THREAD

**Issue by diedrichg** [tidusjar/Ombi#1198]
#### Ombi Version:

V 2.1.1469 

#### Update Branch:

Early Access Preview

#### Media Sever:

Emby

#### Operating System:

Win8.1

#### Mono Version (only if your not on windows)

(Place text here)

#### Problem Description:

Upon original Ombi setup I created the admin. From there I then decided to give one of my Emby users admin access. A few weeks later (today), I decided to remove the admin credentials from my Emby user so I logged in as the original Ombi admin and removed the admin rights from the Emby user. I then logged in as the Emby user and it still had full admin privileges.

**New scenario:**
 I had "Auto Approve Movie Requests" previously enabled for the Emby user. This caused the request to be made at a lower resolution. So I went in as the Ombi admin and removed the request. I then downgraded the Emby user to basic user and turned off "Auto Approve Movie Requests". I then logged in as the Emby user and attempted to make a request for the same movie (as before with admin rights), I was given a error:
 `Could not add movie, please contract your administrator`
The logs show:
`This movie has already been added	Ombi.Core.MovieSender`

1. Emby user (admin rights) made a movie request for Movie ABC123 which was Auto Approved and at a lower resolution than desired.
2. Ombi admin logged in and removed Emby user's admin rights and took off Auto Approve as well as removed the request for Movie ABC123.
3. Ombi admin logged out
4. Emby user (now normal user and no Auto Approve) logged in and made the same request for Movie ABC123 but was given the "Could not add movie, please contract your administrator" error. The Movie ABC123 was not added and the log shows "This movie has already been added	Ombi.Core.MovieSender".
5. Emby user (without admin rights) still has full admin access and rights.

**Comment by diedrichg** [comment]
Update:
I stopped the Ombi service and then started it. I logged in as the Emby user (scenario above) and still had admin rights. 

However, the requested movie was now marked as Requested. I don't know if that was realized by Ombi or if Ombi scanned Radarr and saw that it was in the Radarr Wanted list.

**Comment by diedrichg** [comment]
Update:
I stopped the Ombi service and then started it. I logged in as the Emby user (scenario above) and still had admin rights. 

However, the requested movie was now marked as Requested. I don't know if that was realized by Ombi or if Ombi scanned Radarr and saw that it was in the Radarr Wanted list.

BUT, the movie is not in the Requests list in Ombi! It only shows as Requested when doing a search.

**Comment by SuperPotatoMen** [comment]
Hi thank you for you report. Auto Approve Movie Requests is not a admin rights right.
If a user doesn't have the permission to watch other requests we show only a Requested button and not the item on the requests page.

If the task caching emby didn't run yet (ombi doesn't know it is available) we show a error to the end user"Could not add movie, please contract your administrator" This is a knowing issue and will be fixed soon.

The reason you can't request it again is beceause Ombi found the request and a pending item on your Radarr/cp/watcher setup.

**Comment by diedrichg** [comment]
The original post was lengthy, but the issue still remains: give a user admin rights. Let them login and then logout. Then take away admin rights. Logon again as the user and the admin rights remain.

**Comment by SuperPotatoMen** [comment]
What do you mean with the admin rights?

**Comment by SuperPotatoMen** [comment]
And how are you know he has still admin rights?

**Comment by diedrichg** [comment]
![2017-02-28 09_11_02-ombi](https://cloud.githubusercontent.com/assets/9559987/23408612/a31e5422-fd96-11e6-8016-8a37dcab6be5.png)

**Comment by tidusjar** [comment]
When you created the admin account (in the wizard) it does state not to use the same name as your main account on Emby/Plex... This is why you are probably seeing this.

**Comment by diedrichg** [comment]
Initial setup:
1. Admin diedrichg was created.
2. Emby user Diedrich was imported
3. Emby user Diedrich was later given Administrator
4. I decided to revoke Administrator from Diedrich so I logged in as diedrichg and removed Administrator from Diedrich
5. Logged in as Diedrich and he still has Administrator privileges. 

Try it. You'll see. Give a user Admin and then take it away. Admin will stick and not revert.
